components/HomePage/InfoSection.tsx

```diff
@@ -1,21 +1,24 @@
 import styled from 'styled-components'
 
-//todo: add dynamic content based on database class descriptions
-export default function InfoSection({ id, imgStart }) {
+interface InfoSectionProps {
+  id?: string
+  imgStart: boolean
```

Comment:
As this is a boolean it should be optional `imgStart?: boolean`

Reply:
its not optional, it determines the layout of the section

Reply:
So it depends how you have your `InfoSection` component setup.

If you always pass a `true` or `false` than it's fine, because it receives a props
so:
`<InfoSection imgStart={false}  />`

but if your component is like this:
`<InfoSection imgStart  />` <-- true
`<InfoSection  />` <-- false

Than typescript sees `imgStart` as an optional prop because it's not set.
So if you always set a boolean, than you can leave it as it is